src/sonarwhal-theme/helper/index.js

```diff
@@ -274,6 +275,25 @@ module.exports = function () {
         isPending: (status) => {
             return status === jobStatus.pending;
         },
+        linkify: (msg) => {
+            const regex = /(https?:\/\/[a-zA-Z0-9.\\/?:@\-_=#]+\.[a-zA-Z0-9&.\\/?:@-_=#]*)\s[a-zA-Z]/g;
+            // Modified use of regular expression in https://stackoverflow.com/a/39220764
+            // Should match:
+            // jQuery@2.1.4 has 2 known vulnerabilities (1 medium, 1 low). See https://snyk.io/vuln/npm:jquery for more information.
+            // AngularJS@1.4.9 has 3 known vulnerabilities (3 high). See https://snyk.io/vuln/npm:angular for more information.
+            // Shouldn't match (shortened url):
+            // File https://www.odysys.com/ … hedule-Your-Demo-Now.png could be around 37.73kB (78%) smaller.
+            const match = regex.exec(msg);
+
+            if (!match) {
+                return msg;
+            }
+
+            const urlMatch = Handlebars.Utils.escapeExpression(match.pop());
```

Comment:
what are you escaping here? There's probably another way of doing it rather than with `escapeExpression`.

Reply:
Based on http://handlebarsjs.com/#html-escaping, it seems that it's safer to manually escape the string before labeling it  as a `safestring`? Is there a result to not use `escapeExpression`?

Reply:
It's just that `requiring` the whole package seems a bit overkill, but we can probably review that at another time.